.github/workflows/Runtests.yml

```diff
@@ -2,10 +2,11 @@ name: Runtests
 on: [push, pull_request]
 jobs:
   test:
+    name: "Runtests ${{matrix.julia-version}}"
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ['1.6']
+        julia-version: ['1.6', '1']
```

Comment:
Test on '1.6' since it is long-term stable and test on the latest stable `1`, what do you think @ATell-SoundTheory ?